.env

```diff
@@ -0,0 +1,2 @@
+CONFIG=  # Here user can key in his own config.
+local=zh  # en or zh
```

Comment:
Just an FYI that this setting won't do anything as you don't have any translatable strings in your code. Shouldn't break anything and future proofs if you want to add translations later (might be worth doing given this is a translation plugin :P )